python/pycryptodome/security/insecure-cipher-algorithm-blowfish.py

```diff
@@ -25,10 +25,12 @@
 bs = pycrypto_blowfish.block_size
 # ruleid:insecure-cipher-algorithm-blowfish
 cipher = pycrypto_blowfish.new(key, pycrypto_blowfish.MODE_CBC, iv)
+# deepruleid:insecure-cipher-algorithm-blowfish
 msg = iv + cipher.encrypt(plaintext + padding)
```

Comment:
Why is symbolic prop on for this rule?
https://github.com/semgrep/semgrep-rules/blob/de3f55438bd280eafedcae0ae4daae1ce61d89c0/python/pycryptodome/security/insecure-cipher-algorithm-blowfish.yaml#L30-L31